samples/msal-browser-samples/HybridSample/views/client-redirect.hbs

```diff
@@ -0,0 +1,98 @@
+<a href="/">Go Back</a><br />
+<a href="/auth/logout">Logout</a>
+
+<script src="/auth/lib/msal-browser.js"></script>
+<script type="text/javascript">
+    const msalInstance = new msal.PublicClientApplication({
+        auth: {
+            clientId: "{{clientId}}",
+            redirectUri: "http://localhost:3000/auth/client-redirect",
+            authority: "https://login.microsoftonline.com/common"
+        }
+    })
+
+    // Only handle hash when in top frame
+    if (window === window.top) {
+        msalInstance.handleRedirectPromise()
+            .then(result => {
+                if (result) {
+                    console.log('MSAL: Returning from login');
+                    return result;
+                }
+                
+                const sid = "{{sid}}";
+                const code = "{{code}}";
+                const preferredUsername = "{{preferredUsername}}";
+                const loginHint = "{{loginHint}}";
+
+                const scopes = [ "user.read" ];
+                const tokenQueryParameters = {
+                    dc: "ESTS-PUB-WUS2-AZ1-FD000-TEST1",
```

Comment:
Do we still need this or has this been rolled out?

Reply:
Still needed for a couple weeks.